client/src/Clusters/TopCharts/TopCharts.js

```diff
@@ -0,0 +1,202 @@
+import emptyAppImage from './Images/emptyAppImage.png';
+import facebookLogo from './Images/facebookLogo.svg';
+import pandoraLogo from './Images/pandoraLogo.png';
+import myWeatherLogo from './Images/myWeatherLogo.png';
+import React, { Component } from 'react';
+import './SingleAppContainer.js';
+import SingleAppContainer from './SingleAppContainer.js';
+import './TopChartsStyle.css';
+import '../styles.css';
+import wearOSLogo from './Images/wearOSLogo.png';
+
+ 
+class TopCharts extends Component {
+  render() {
+    const makeColumn = item => (
+        <div key={item.title}>
```

Comment:
unfortunately title isn't unique so this can cause rendering issues if theres 2 items with the same key.